r2/r2/controllers/api.py

```diff
@@ -1175,6 +1175,10 @@ def POST_friend(self, form, jquery, friend,
             table = jquery("." + type + "-table").show().find("table")
             table.insert_table_rows(user_row, index=index)
             table.find(".notfound").hide()
+            if type == 'moderator':
```

Comment:
Can you add a comment above this saying what it's doing? Doing UI code in here is ugly, so it's nice to see what it's for from comments instead of needing to puzzle it out.